src/components/CurrentFlashcard.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import Popup from "./Popup";
+import '../styles/CurrentFlashcard.css'
+import '../styles/Button.css'
+import DeleteButton from "./DeleteButton";
+
+const CurrentFlashcard = ({ setCardActive, index, flashcards }) => {
+    const [definitionActive, setDefinitionActive] = useState(false)
+    const [currentCardIndex, setCurrentCardIndex] = useState(index)
+
+    const { name, definition } = flashcards[currentCardIndex]
+
+    const changeNameHandler = () => {
+        setDefinitionActive(state => !state)
+    }
+
+    const arrowHandler = sign => {
+        if (sign === '<') {
```

Comment:
Może zamiast opierać się o znak to lepiej napisać osobne funkcje - będzie czytelniej tj. `prevHandler` oraz `nextHandler`